Enroll.html

```diff
@@ -9,6 +9,54 @@
     href="Enroll.css">
 </head>
 <body>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+<body>
+    <img src="maskot.jpg" alt="maskot" height="150" width="150">
+    <div class="box-2">
+
+        <h1>Riverdale</h1>
+        <h1>Apes</h1>
+        <h6>208-249-9096</h6>
+   
+
+    </div>
+
+
+    <div class="box-1">
+        <a class="nav-link active" aria-current="page" href="resume.html"><b>Home</b></a>
+        <a class="nav-link active" aria-current="page" href="resume.html"><b>staff</b></a>
+        <a class="nav-link active" aria-current="page" href="resume.html"><b>sports and merchandice</b></a>
+        <a class="nav-link active" aria-current="page" href="resume.html"><b>counseling</b></a>
+        <a class="nav-link active" aria-current="page" href="resume.html"><b>enrollment form</b></a>
+</div>
+
+
+
+
+
+
+
+<hr>
+<div class="box-3">

```

Comment:
Lines 41 to 47 should be at the bottom of the page

Reply:
yes